source/db/en-projects.json

```diff
@@ -889,6 +889,7 @@
     "wikipedia_url": "",
     "protocols": [
       "XMPP",
+      "OMEMO",
```

Comment:
Pretty sure there will be outdated English localizations in other language files. You'll need to adjust those too or (even better) just remove them entirely.